src/openforms/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -4,7 +4,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Open Forms\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-11-29 15:40+0100\n"
+"POT-Creation-Date: 2022-12-06 14:09+0100\n"
```

Comment:
Please revert all changes to the `.po` file(s) - these do not belong in feature pull requests.

Translations are managed shortly before a release and this is done exclusively by Joeri or me.